### PR TITLE
Anselmo ioreadhit

### PR DESCRIPTION
**Pull request by amerega**
New ioread to read old data after changes in the Hit structure (from fSummedADC to fROISummedADC and fHitSummedADC)

### CODE REVIEW

File: lardataobj/RecoBase/classes_def.xml [hunk 1]
```diff
@@ -234,6 +234,33 @@
   <class name="art::Wrapper< std::vector< std::vector<recob::TrackFitHitInfo>>>"/>
 
   <!-- io rules for types in this file -->
+
+<ioread                                                                                                                                                                                        
+      version="[16]"
```
Comment (knoepfel) [inline lardataobj/RecoBase/classes_def.xml]: Should be `version=[-16]`, not just `version=[16]`.  (The hyphen means that all versions before 16 should also have the IO rule applied to them.)

File: lardataobj/RecoBase/classes_def.xml [hunk 2]
```diff
@@ -234,6 +234,33 @@
...
+      target="fHitSummedADC"                                                                                                                                                                     
+      include="lardataobj/RecoBase/Hit.h"                                                                                                                                                        
+  >                                                                                                                                                                                              
+  <![CDATA[                                                                                                                                                                                      
+           fHitSummedADC = 0;                                                                                                                                                                    
+  ]]>                                                                                                                                                                                            
+ </ioread>
```
Comment (knoepfel) [inline lardataobj/RecoBase/classes_def.xml]: This IO rule can be removed because ROOT will automatically default construct new data members with 0.

**Review by knoepfel — COMMENTED** [review]
Thanks, @amerega, for this PR!  Looks pretty good.  Just a couple comments.

File: lardataobj/RecoBase/classes_def.xml (hunk 2 above)
Comment (amerega) [reply lardataobj/RecoBase/classes_def.xml]: Hi Kyle, thanks for the feedback, I committed a corrected version.

**Comment by knoepfel** [comment]
trigger build

**Comment by lgarren** [comment]
trigger build

**Comment by lgarren** [comment]
trigger build

**Comment by lgarren** [comment]
trigger build

**Comment by lgarren** [comment]
trigger build

**Comment by lgarren** [comment]
trigger build

Hoping this does not fail at `setup_environment_cvmfs` stage.

**Comment by lgarren** [comment]
approve

**Pull request merged**